pybossa/view/projects.py

```diff
@@ -3504,6 +3504,35 @@ def del_coowner(short_name, user_name=None):
     return abort(404)
 
 
+@blueprint.route('/<short_name>/ownership_id', methods=['GET', 'PUT'])
+@login_required
+def ownership_id(short_name):
```

Comment:
Can the existing project info endpoint `/api/project/<project_id>` be used instead of a new endpoint specific for ownership id? If so, this method `ownership_id` can be removed.

Reply:
I believe we still need this endpoint to fetch the ownership_id from the pybossa config. Since the frontend consumer is vue.js, we aren't able to set the value without an API call.

Reply:
It may be possible to read the ownership_id label from the config in Jinja code (in the HTML template) by using a hidden div or span element. Then vuejs can query the div value to retrieve the label.

```html
<div data-title="{ current_app.config.get('OWNERSHIP_ID_TITLE', 'Ownership ID') }">
   <!-- form field elements for ownership row -->
</div>
```